src/Parser/Core/Modules/Others/DistanceMoved.js

```diff
@@ -66,13 +74,13 @@ class DistanceMoved extends Analyzer {
   }
 
   statistic() {
-    debug && console.log(`Total distance moved: ${this.totalDistanceMoved} yds`);
+    debug && console.log(`Time spent moving: ${this.timeSpentMoving / 1000} s, Total distance moved: ${this.totalDistanceMoved} yds`);
     return (
       <SmallStatisticBox
         icon={<Icon icon="spell_fire_burningspeed" />}
-        value={`≈${formatThousands(this.totalDistanceMoved)} yards`}
-        label="Distance moved"
-        tooltip={`≈${formatThousands(this.totalDistanceMoved / (this.owner.fightDuration / 1000) * 60)} yards per minute. Consider this when analyzing the fight, as some fights require more movement than others. Unnecessary movement can result in a DPS/HPS loss.`}
+        value={`≈${formatPercentage(this.timeSpentMoving / (this.owner.fightDuration))}%`}
+        label="Fight spent moving"
```

Comment:
Shouldn't this be "Time spent moving"?

Reply:
I guess it makes a little more sense if read with the value, like "20% Fight spent moving"...

Reply:
I agree, sounds better. Addressed.